docs-site/content/0.19.0/guide/features/raft.md

```diff
@@ -0,0 +1,5 @@
+# Raft Based Clustering
+
+High availability is essential for production environments. Typesense uses [Raft algorithm](https://raft.github.io/) to create a highly available cluster with more than one Typesense servers. Based on raft consensus, to handle 1-node failure, you need to create a cluster with 3 nodes. Similarly for 2-node failure, you need to create a 5 node cluster. Note that adding more nodes will also add write latencies. 
```

Comment:
> uses Raft algorithm

Correction: "uses **the** Raft Consensus Algorithm" 

> Based on raft consensus, to handle 1-node failure, you need to create a cluster with 3 nodes. Similarly for 2-node failure, you need to create a 5 node cluster.

Correction:

With Raft, you need to create a cluster of 3 nodes to tolerate single node failures. If you wish to handle 2-node failures, then you need a minimum of 5 nodes in the cluster. 

> Note that adding more nodes will also add write latencies. 

Correction:

Note that adding more nodes will also increase write latencies.